### ISSUE TITLE
datalad-container namespace is not picked up due to missing required `intermixed` 

### ISSUE THREAD

**Issue by yarikoptic** [datalad/datalad#7678]
```
❯ DATALAD_LOG_LEVEL=1 datalad  containers-list
[Level 5] Instantiating ssh manager 
[Level 5] Done importing main __init__ 
[Level 5] Importing cli.main 
[Level 5] Importing support.network 
[Level 5] Done importing support.network 
[DEBUG  ] Retro-fitted GitRepo with deprecated add_submodule 
[DEBUG  ] Retro-fitted GitRepo with deprecated deinit_submodule 
[DEBUG  ] Retro-fitted GitRepo with deprecated update_submodule 
[Level 1] parseParameters: Given "exclude_special_remotes: bool, optional
  if True, don't return annex special remotes", we split into [('exclude_special_remotes', "exclude_special_remotes: bool, optional\n  if True, don't return annex special remotes")] 
[Level 1] parseParameters: Given "with_urls_only : bool, optional
  return only remotes which have urls", we split into [('with_urls_only', 'with_urls_only : bool, optional\n  return only remotes which have urls')] 
[Level 5] Importing dataset 
[Level 5] Done importing dataset 
[Level 5] Importing datalad.interface.results 
[Level 5] Done importing datalad.interface.results 
[Level 5] Done importing cli.main 
[Level 5] Starting main(['/home/yoh/proj/datalad/datalad-container/venv/3.12/bin/datalad', 'containers-list']) 
[Level 5] Starting to setup_parser 
[Level 8] ArgumentParser._parse_known_args() missing 1 required positional argument: 'intermixed' [parser.py:single_subparser_possible:356] 
[DEBUG  ] Early parsing failed with TypeError(ArgumentParser._parse_known_args() missing 1 required positional argument: 'intermixed') 
```

the cost of us using private interfaces and python 3.12.7 (what I have) having it changed -- @mih will solve it all with new salads and cores!

**Comment by yarikoptic-gitmate** [comment]
Issue fixed in [`1.1.4`](https://github.com/datalad/datalad/releases/tag/1.1.4)

**Comment by yarikoptic-gitmate** [comment]
Issue fixed in [`1.1.4`](https://github.com/datalad/datalad/releases/tag/1.1.4)
